src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -198,7 +196,6 @@ const SectionLink = styled.a`
   min-height: 26px;
   padding-bottom: 5px;
   padding-left: 15px;
-  cursor: pointer;
```

Comment:
do you know why was it here before?

Reply:
I think that probably for some reason this link didn't have `href` attribute in some of the previous versions and because of that it's cursor displayed incorrectly. But now all links have `href`'s, so it should be ok to remove it.

Reply:
👍